WordPress/src/main/res/layout/fragment_login_no_sites_empty_view.xml

```diff
@@ -0,0 +1,126 @@
+            android:padding="@dimen/margin_extra_large"
+            android:src="@drawable/img_jetpack_empty_state"
+            app:layout_constraintBottom_toTopOf="@+id/login_error_message_title"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_empty_view_user_section"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/login_error_message_title"
+            style="@style/ActionableEmptyStateTitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/margin_medium"
+            android:gravity="center_horizontal"
+            android:lineSpacingExtra="@dimen/login_empty_view_line_spacing"
+            android:text="@string/login_no_jetpack_sites"
+            app:layout_constraintBottom_toTopOf="@+id/login_error_message_text"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_empty_view_illustration"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/login_error_message_text"
+            style="@style/ActionableEmptyStateSubtitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal"
+            android:lineSpacingExtra="@dimen/login_empty_view_line_spacing"
+            android:text="@string/login_no_jetpack_sites_error_message"
+            android:paddingBottom="@dimen/margin_extra_large"
+            app:layout_constraintBottom_toTopOf="@+id/button_primary"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_error_message_title"/>
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/button_primary"
+            style="@style/Widget.LoginFlow.Button.Primary"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:text="@string/login_empty_view_see_instructions"
+            android:textAllCaps="false"
+            app:layout_constraintBottom_toTopOf="@+id/button_secondary"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_error_message_text" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/button_secondary"
+            style="@style/Widget.LoginFlow.Button.Secondary"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/margin_extra_large"
```

Comment:
Question (❓): Is this bottom margin needed here?

Reply:
TBH, I couldn't tell you what I added margin/padding two weeks ago. Going to wait for your answer re: specs before adjusting anything else in the view. Thanks.